### PR TITLE
LPS-162006 Set Failed status on Notification Queue Entry

### PR DESCRIPTION
**Pull request by mateussgn**
Related issue: https://issues.liferay.com/browse/LPS-162006

### CODE REVIEW

**Comment by mateussgn** [comment]
ci:test:relevant

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -213,16 +213,24 @@ public void sendNotificationQueueEntries() throws PortalException {
...
 					notificationQueueEntry.getNotificationQueueEntryId(), true);
 			}
-		}
-		catch (PortalException portalException) {
-			throw portalException;
-		}
-		catch (Exception exception) {
-			if (_log.isDebugEnabled()) {
-				_log.debug(exception);
+			catch (PortalException portalException) {
```
Comment (rodrigopaulino) [inline modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java:216]: This catch is not needed.

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java [hunk 2]
```diff
@@ -252,6 +260,14 @@ public NotificationQueueEntry updateSent(
 		return notificationQueueEntryPersistence.update(notificationQueueEntry);
 	}
 
+	public NotificationQueueEntry updateStatus(
```
Comment (rodrigopaulino) [inline modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java:263]: You don't need this method. Also, you can just call `notificationQueueEntry.persist()` to update the entry. ;)

**Review by rodrigopaulino — CHANGES_REQUESTED** [review]
Hey, @mateussgn. Could you take care of these changes before going to the next story?
Thx

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java (hunk 1 above)
Comment (mateussgn) [reply modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java:216]: Well observed! Thank you!

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java (hunk 2 above)
Comment (mateussgn) [reply modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java:263]: It was a valuable tip!
Besides, I forgot to run `gw buildService` after the method creation. Won't need it anymore.

**Comment by mateussgn** [comment]
ci:test:sf

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:stop

**Comment by mateussgn** [comment]
ci:test:sf

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:test:sf

**Comment by michaelhashimoto** [comment]
ci:reevaluate:1347401_2731

**Review by rodrigopaulino — APPROVED** [review]
LGTM

**Comment by rodrigopaulino** [comment]
ci:stop

**Comment by mateussgn** [comment]
Directly sent to @brianchandotcom here: https://github.com/brianchandotcom/liferay-portal/pull/122639

**Pull request closed without merging**